src/main.c

```diff
@@ -152,6 +179,11 @@ int main(void) {
   Serial_Init(SERIAL_BAUDRATE);
   CAN_Init(CAN_BAUDRATE);
   ADC_Init();
+  Timer_Init();
+
+  Set_Interrupt_Priorities();
```

Comment:
why did you make timer init and set interrupt priorities separate (and in separate files??). seems like they both deal with hardware-specific timer initialization, although i could be missing something